### DISCUSSION TITLE
Eslint configuration for Breeze and Jetstream

### DISCUSSION THREAD

**Discussion by mahdiaslami** [laravel/framework#46397] [Ideas]
Excuse me. Is it possible to share the Eslint config used in Breeze or Jetstream packages somewhere? It can be helpful for us to have the same config because there is a lot of component in these packages and every time we should re-format them to our config and sharing this config can resolve this effort.

Thank you.

**Comment by hfoletto** [comment]
Hey @mahdyaslami what did you end up using?
